R/analytical_functions.R

```diff
@@ -43,6 +43,13 @@
 #'
 #' Default setup is 1/3 for every age class <= 5 life years, and 1/2 for the others.
 #'
+#' @param option_spline integer, optional. If > 0, values for adults will be
+#' interpolated by a monotonic cubic spline. Usual options will by '10' or '20' which
+#' will interpolate the values for individuals of an age of 20 or older by 10- or 20-
+#' year cumulated values. To be used carefully, as diagnostic features of the life
+#' table might be smoothed and essentially removed. Only available when the methods
+#' 'Standard' or 'Equal5' in prep.life.table have been chosen.
```

Comment:
This option should not soft-depend on an obscure (*obscure for the user*) option in the `prep` function. The `prep` function is not necessary to use `life.table()`. Instead the conditions to use this option should be explained here directly. I also suggest to add a check at the beginning of the `if (length(option_spline) > 0) {...}` block to test if the condition is met.

Reply:
A check like this could look like this: 

```
if (!condition) {
  stop("The condition is not fulfilled. Please take a look at ?life.table to determine how your input data should look like for the option_spline option.")
}
```